Node/src/dialogs.ts

```diff
@@ -111,7 +112,7 @@ export class AuthDialog extends builder.Dialog {
                 msg = new builder.Message(session)
                     .attachments([
                         new builder.SigninCard(session)
-                            .text("connect_prompt")
+                        .text(args.providerId === 'azuread-openidconnect' ? args.skypeSignIn : "connect_prompt")
```

Comment:
I dont know that you can just call args.providerId.   Try doing what they did above for the buttonUrl -- `opt.buttonUrl`

Reply:
same for opt.skypeSignIn